tests/storage/snapshot.go

```diff
@@ -733,108 +717,6 @@ var _ = Describe(SIG("VirtualMachineSnapshot Tests", func() {
 				})))
 			})
 
-			It("Calling Velero hooks should freeze/unfreeze VM", func() {
```

Comment:
I think this is worth having as a functional test, any reason to remove it?

Reply:
checking that this actually freezes/unfreezes the fs tests the libvirt functionality rather then the binary, the UT makes sure the freeze call is being done eventually and that is the actual thing we want to know that happens, so that covers what we want to test.
I think creating a vm and calling the binary in the pod is redundant just for the sake of seeing that the binary calls freeze.
Also if we want to check freeze is being called by specifically the velero hook we need to test it in the kubevirt-velero-plugin repo - that will be an end to end test.